File: 프로그래머스/1/131705. 삼총사/삼총사.py
def solution(number):
    answer = 0
    pointa = 0
    pointa = 1
    pointb = 2
    
    for i in range(0,len(number)):
        pointa = i
        for j in range(i+1,len(number)) :
            pointb = j
            for k in range(j+1,len(number)) :
                pointc = k
                if number[i] + number[j] + number[k] == 0 :
                    answer +=1
                print(i,j,k)                  
    return answer

File: 프로그래머스/1/131705. 삼총사/삼총사.java
class Solution {
    public int solution(int[] number) {
        int answer = 0;
        int numlen = number.length;
        
        for (int i=0; i<numlen-2; i++){
            for(int j=i+1; j<numlen-1; j++){
                for(int k=j+1; k<numlen; k++){
                    if( number[i] + number[j] + number[k] ==0){
                        answer +=1;
                    }
                }
            }
        }
        return answer;
    }
}